polkadot/parachain/src/primitives.rs

```diff
@@ -199,13 +199,14 @@ impl From<i32> for Id {
 	}
 }
 
-const USER_INDEX_START: u32 = 1000;
+// System parachain ID is considered `< 2000`.
+const SYSTEM_INDEX_END: u32 = 1999;
 const PUBLIC_INDEX_START: u32 = 2000;
 
 /// The ID of the first user (non-system) parachain.
-pub const LOWEST_USER_ID: Id = Id(USER_INDEX_START);
+pub const LOWEST_USER_ID: Id = Id(PUBLIC_INDEX_START);
```

Comment:
I was not able to find any reference on github other than forks of Polkadot: 
https://github.com/search?q=LOWEST_USER_ID+language%3ARust&type=code&ref=advsearch
(let me know if you know how to filter out forks in the search query)

so it should be fine to remove